### PR TITLE
Tyk releng update [MASTER]

### PR DESCRIPTION
**Pull request by ermirizio**
Result of gromit templating update along with sync-automation workflow

### CODE REVIEW

**Comment by Tyk-ITS** [comment]
**API tests result: success** :white_check_mark: 
Branch used: refs/pull/5285/merge
Commit:  
Triggered by: pull_request (@ermirizio)
[Execution page](https://github.com/TykTechnologies/tyk/actions/runs/5477898964)

**Comment by Tyk-ITS** [comment]
**API tests result: success** :white_check_mark: 
Branch used: refs/pull/5285/merge
Commit: bcc2c6922672fb61f19110a452218e75f455db39 
Triggered by: pull_request (@ermirizio)
[Execution page](https://github.com/TykTechnologies/tyk/actions/runs/5478894513)

File: .github/workflows/api-tests.yml [hunk 1]
```diff
@@ -60,22 +60,23 @@ jobs:
         repository: TykTechnologies/tyk-automated-tests
         token: ${{ secrets.ORG_GH_TOKEN }}
         path: tyk-automated-tests
-        ref: ${{ steps.check_test_branch.outputs.branch	 }}
+        ref: ${{ steps.check_test_branch.outputs.branch   }}
```
Comment (titpetric) [inline .github/workflows/api-tests.yml:63]: fix whitespace

File: .github/workflows/api-tests.yml [hunk 2]
```diff
@@ -117,7 +118,11 @@ jobs:
       run: |
         while [[ "$(curl -s -o /dev/null -w ''%{http_code}'' localhost:3000/hello/)" != "200" ]]; do sleep 60 && echo "waiting for dashboard '$(date +"%T")'"; done
       timeout-minutes: 15    
-    
+
+    - name: Getting tyk logs on failure
+      if: ${{ failure() }}      
+      run: docker logs tyk-gateway
```
Comment (titpetric) [inline .github/workflows/api-tests.yml:124]:
```suggestion
      run: docker-compose -f ci/ci_testing_env.yml logs
```

File: .github/workflows/api-tests.yml [hunk 3]
```diff
@@ -140,16 +145,18 @@ jobs:
          name: api-test-report
          path: ./tyk-automated-tests/reports/
 
+
     - name: Notify slack
       if: ${{ always() }}
       uses: rtCamp/action-slack-notify@v2
       env:
-        SLACK_WEBHOOK: ${{ secrets.GW_SLACK_WEBHOOK }}
+        SLACK_WEBHOOK: ${{ secrets.GW_SLACK_WEBHOOK}}
```
Comment (titpetric) [inline .github/workflows/api-tests.yml:153]: trailing whitespace missing (consistency)

**Comment by Tyk-ITS** [comment]
**API tests result: skipped** :no_entry_sign: 
Branch used: refs/pull/5285/merge
Commit: 579cd939fd4c93acfa6c237a5375e6443e738a27 
Triggered by: pull_request (@ermirizio)
[Execution page](https://github.com/TykTechnologies/tyk/actions/runs/5490186205)

**Comment by Tyk-ITS** [comment]
**API tests result: failure** :no_entry_sign: 
Branch used: refs/pull/5285/merge
Commit: 970705365c93eebfde54e2a18d7ae27a182ee762 
Triggered by: pull_request (@ermirizio)
[Execution page](https://github.com/TykTechnologies/tyk/actions/runs/5490117752)

**Comment by Tyk-ITS** [comment]
**API tests result: failure** :no_entry_sign:
Branch used: refs/pull/5285/merge
Commit: c2fcc0c7e60fd378b3216943a8dd85f93939111d 
Triggered by: pull_request (@ermirizio)
[Execution page](https://github.com/TykTechnologies/tyk/actions/runs/5490234078)

**Comment by Tyk-ITS** [comment]
**API tests result: failure** :no_entry_sign:
Branch used: refs/pull/5285/merge
Commit: 0eea406893dda13d296bbf3015d46fe36bd1924d 
Triggered by: pull_request (@ermirizio)
[Execution page](https://github.com/TykTechnologies/tyk/actions/runs/5490319863)

**Comment by Tyk-ITS** [comment]
**API tests result: failure** :no_entry_sign:
Branch used: refs/pull/5285/merge
Commit: c74cbde3542e3d02b015545afb6dca9d1e1826b9 
Triggered by: pull_request (@ermirizio)
[Execution page](https://github.com/TykTechnologies/tyk/actions/runs/5490490988)

**Comment by Tyk-ITS** [comment]
**API tests result: failure** :no_entry_sign:
Branch used: refs/pull/5285/merge
Commit: dd0afa4b3ac6198284573218c234d5d8dd5f87e3 
Triggered by: pull_request (@ermirizio)
[Execution page](https://github.com/TykTechnologies/tyk/actions/runs/5490286916)

**Comment by Tyk-ITS** [comment]
**API tests result: success** :white_check_mark:
Branch used: refs/pull/5285/merge
Commit: 9cf592f89b7d96a7a5fb98bef512af54dcc226dc 
Triggered by: pull_request (@ermirizio)
[Execution page](https://github.com/TykTechnologies/tyk/actions/runs/5510581388)

**Comment by Tyk-ITS** [comment]
**API tests result: success** :white_check_mark:
Branch used: refs/pull/5285/merge
Commit: c6df6bb409a7506f9c2f7b4b55c517bbdcc712d3 
Triggered by: pull_request (@ermirizio)
[Execution page](https://github.com/TykTechnologies/tyk/actions/runs/5510690572)

**Comment by Tyk-ITS** [comment]
**API tests result: success** :white_check_mark:
Branch used: refs/pull/5285/merge
Commit: 8a5edcf6320ffe3efdf8f3b4a41ed8ea836a6490 
Triggered by: pull_request (@ermirizio)
[Execution page](https://github.com/TykTechnologies/tyk/actions/runs/5510918544)

**Comment by Tyk-ITS** [comment]
**API tests result: success** :white_check_mark:
Branch used: refs/pull/5285/merge
Commit: 8e63d2fa9d416e670e208948377f0a54ffaf83ea 
Triggered by: pull_request (@ermirizio)
[Execution page](https://github.com/TykTechnologies/tyk/actions/runs/5580812805)

**Comment by Tyk-ITS** [comment]
**API tests result: success** :white_check_mark:
Branch used: refs/pull/5285/merge
Commit: 53ccbcc7e39b3add8e10ea3fecb2aa353cad63e1 
Triggered by: pull_request (@ermirizio)
[Execution page](https://github.com/TykTechnologies/tyk/actions/runs/5651040209)

**Comment by Tyk-ITS** [comment]
**API tests result: success** :white_check_mark:
Branch used: refs/pull/5285/merge
Commit: 876da1aa05bb3704c706d88e909467d1f1015bae 
Triggered by: pull_request (@ermirizio)
[Execution page](https://github.com/TykTechnologies/tyk/actions/runs/5651093683)

**Pull request closed without merging**